airbyte-integrations/connectors/source-iterable/acceptance-test-config.yml

```diff
@@ -80,6 +80,18 @@ acceptance_tests:
         bypass_reason: "Can not populate; need messaging service configured"
       - name: "web_push_send_skip"
         bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_open"
+        bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_bounce"
+        bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_click"
+        bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_send"
+        bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_subscribe"
+        bypass_reason: "Can not populate; need messaging service configured"
+      - name: "email_unsubscribe"
```

Comment:
As I saw it, we had this data before. Why can't we fill this stream now.

Reply:
@igrankova @ycherniaiev Do you have any information about this?

Reply:
I will merge this for now, once we have these updated, another PR will be created to remove email_* like streams from `empty streams`. @lazebnyi